### PR TITLE
New Docker Compose for Openclassify with PHP 8.2

### PR DESCRIPTION
**Pull request by fatihalp**

### CODE REVIEW

**Review by MostafaMoradii — APPROVED** [review]

File: install.sh [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    fi
+fi
+
+cp .env-sail .env
+docker compose up  
+#php artisan migrate --all-addons --force
+#docker exec -it oc_php php artisan install --ready
+
+#composer config http-basic.abc.aaa.com openclassify pass
+#composer config repositories.repo-name composer https://abc.aaa.com
```
Comment (muammertopcu) [inline install.sh:47]: Do we need these lines of code? If they are necessary, why did we leave them as comments? I think they might have been forgotten.

File: install.sh [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+echo "================================================"
+echo "----------------- Openclassify -----------------"
+echo "----- Automated install script with Docker -----"
+echo "================================================"
+
+if [[ $(which docker) && $(docker --version) ]]; then
+    docker --version
+
+    #if ubuntu install docker
```
Comment (erhanask) [inline install.sh:11]: # If Ubuntu, check if Docker is running

Comment (fatihalp) [reply install.sh:11]: it's acutally todo. it's already starting docker if OS is Ubuntu

File: install.sh (hunk 1 above)
Comment (fatihalp) [reply install.sh:47]: that's right. I'll move them to docs as Usefull Commands

**Review by emrullahardc — APPROVED** [review]

**Review by emrullahardc — APPROVED** [review]

**Pull request merged**